app/screens/custom_status/custom_status_modal.tsx

```diff
@@ -232,6 +266,15 @@ class CustomStatusModal extends PureComponent<Props, State> {
                     secureTextEntry={false}
                     keyboardAppearance={getKeyboardAppearanceFromTheme(theme)}
                 />
+                {customStatusEmoji}
+                {isStatusSet ? (
```

Comment:
maybe we can do something like this instead
```suggestion
                {isStatusSet && (
```